src/schoology-extractor/schoology_extractor/helpers/export_data.py

```diff
@@ -21,10 +22,11 @@ def to_csv(data: list, output_path: str):
     assert isinstance(data, list)
     assert isinstance(output_path, str)
 
+    logging.info("Generating CSV File")
     df = pd.DataFrame(data)
 
     df.to_csv(output_path, index=False)
-    print(f"The file has been generated => {output_path}")
+    logging.info(f"The file has been generated => {output_path}")
```

Comment:
As a matter of habit, should we use the `%s` formatting style here?

Reply:
Done